src/main/java/de/kekru/struktogrammeditor/control/Controlling.java

```diff
@@ -13,42 +13,49 @@
 import java.awt.event.WindowEvent;
 import java.awt.event.WindowListener;
 import java.awt.image.BufferedImage;
+import java.awt.print.PrinterException;
+import java.awt.print.PrinterJob;
 import java.io.File;
 import java.io.IOException;
 
 import javax.swing.JCheckBoxMenuItem;
+import javax.swing.JFileChooser;
 import javax.swing.JOptionPane;
 import javax.swing.LookAndFeel;
 import javax.swing.UIManager;
 import javax.swing.UnsupportedLookAndFeelException;
+import javax.swing.filechooser.FileNameExtensionFilter;
+import javax.swing.plaf.nimbus.NimbusLookAndFeel;
 
 import com.sun.java.swing.plaf.motif.MotifLookAndFeel;
-import javax.swing.plaf.nimbus.NimbusLookAndFeel;
 
 import de.kekru.struktogrammeditor.other.Helpers;
 import de.kekru.struktogrammeditor.other.XActionCommands;
+import de.kekru.struktogrammeditor.struktogrammelemente.AnweisungsTyp;
 import de.kekru.struktogrammeditor.struktogrammelemente.StruktogrammElement;
 import de.kekru.struktogrammeditor.view.CodeErzeuger;
 import de.kekru.struktogrammeditor.view.EinstellungsDialog;
 import de.kekru.struktogrammeditor.view.FontChooser;
 import de.kekru.struktogrammeditor.view.GUI;
 import de.kekru.struktogrammeditor.view.ZoomEinstellungen;
 
+//Necessary because NotifyLookAndFeel is now restricted 
+@SuppressWarnings("restriction")
 public class Controlling implements Konstanten, ActionListener, WindowListener, KeyListener {
 
 	private GUI gui;
-	private enum Betriebssysteme {Windows, Mac, Linux};
-
+	
 	public Controlling(String[] params){
 		handleOSSettingsAndLookAndFeel();
 
 		gui = new GUI(this);
 		neuesStruktogramm();		
 
 		if(params != null){
-			for(int i=0; i < params.length; i++){
-				if(new File(params[i]).exists()){
-					openStruktogramm(params[i]);
+			for(int i = 0; i < params.length; i++){
+				File f = new File(params[i]);
+				if(f.exists()){
+					openStruktogramm(f);
 				}
 			}

```

Comment:
Now, when you updated to Java 8, you can do the following ;)

```suggestion
Stream.of(params)
  .filter(File::exists)
  .forEach(this::openStruktogramm);
```

Reply:
It is a little bit more verbose, but I changed it and will commit it later